src/clr-angular/utils/drag-and-drop/providers/drag-dispatcher.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Injectable, NgZone, Renderer2} from "@angular/core";
+import {Observable} from "rxjs/Observable";
+import {Subject} from "rxjs/Subject";
+
+import {DragPosition} from "../interfaces/client-position";
+import {Draggable} from "../interfaces/draggable";
+
+@Injectable()
+export class DragDispatcher {
+    private _listeners: Function[];
```

Comment:
Replace `Function` by the actual type.

Reply:
Are you recommending `() => void` instead of `Function`?

Reply:
Exactly. :) It shows better what it does.